src/avram/queryable.cr

```diff
@@ -151,7 +153,7 @@ module Avram::Queryable(T)
     if query.ordered?
       query
     else
-      query.order_by(:id, :asc)
+      id.asc_order.query
```

Comment:
Type safe, and also adds the table name when ordering

Reply:
nice!